lib/core/src/putUtil.cpp

```diff
@@ -670,14 +670,6 @@ initCondForPut( rcComm_t *conn, rodsEnv *myRodsEnv, rodsArguments_t *rodsArgs,
         }
     }
 
-    if ( rodsArgs->wlock == True ) { // JMC - backport 4604
-        addKeyVal( &dataObjOprInp->condInput, LOCK_TYPE_KW, WRITE_LOCK_TYPE );
```

Comment:
Is anything else using `LOCK_TYPE_KW` or `WRITE_LOCK_TYPE`? If not, let's remove those too.